inc/panels.php

```diff
@@ -214,7 +214,8 @@ function vantage_panels_save_post( $post_id, $post ){
 function vantage_panels_row_style_fields($fields) {
 	if(
 		( !empty($_REQUEST['postId']) && get_post_meta( intval( $_REQUEST['postId'] ), 'vantage_panels_no_legacy', true ) === 'true' ) ||
-		( get_the_ID() && get_post_meta( get_the_ID(), 'vantage_panels_no_legacy', true ) === 'true' )
+		( get_the_ID() && get_post_meta( get_the_ID(), 'vantage_panels_no_legacy', true ) === 'true' ) ||
+		( ! empty( $_REQUEST['page'] ) && $_REQUEST['page'] === 'so_panels_home_page' && get_post_meta( intval( get_option('siteorigin_panels_home_page_id') ), 'vantage_panels_no_legacy', true ) === 'true' )
```

Comment:
Looks fine. Not a blocker, but it's hard to follow the logic here, so I would consider extracting some of these into variables with names and maybe even add some comments if necessary to explain what's going on and why.